utils/iok.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+#
+# Various utilities
+#
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+show_help() {
+  echo "get_full_graph    Print full graph json from CID"
```

Comment:
minor nit but isn't this missing the other commands?

Reply:
ah! nice catch, I'll add those in